package.json

```diff
@@ -116,7 +119,8 @@
 		"@typescript-eslint/parser": "^4.8.2",
 		"babel-loader": "^8.2.5",
 		"core-js": "^3.25.2",
-		"eslint": "^7.27.0",
+		"date-fns": "^2.30.0",
```

Comment:
What package uses `date-fns`?

Reply:
New version of react-day-picker

Reply:
Percy snapshot job was failing without it:
https://app.circleci.com/pipelines/github/purple-technology/phoenix-components/2054/workflows/a1a2f364-762e-4af8-be5d-697923cb9330/jobs/3787